[2022-02-12 17:36:54,985] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_dag.ingest scheduled__2021-01-02T06:00:00+00:00 [queued]>
[2022-02-12 17:36:55,000] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_dag.ingest scheduled__2021-01-02T06:00:00+00:00 [queued]>
[2022-02-12 17:36:55,001] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 17:36:55,001] {taskinstance.py:1239} INFO - Starting attempt 3 of 3
[2022-02-12 17:36:55,001] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 17:36:55,017] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2021-01-02 06:00:00+00:00
[2022-02-12 17:36:55,022] {standard_task_runner.py:52} INFO - Started process 625 to run task
[2022-02-12 17:36:55,028] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'ingest', 'scheduled__2021-01-02T06:00:00+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpadxj17ad', '--error-file', '/tmp/tmpfy6_myz3']
[2022-02-12 17:36:55,030] {standard_task_runner.py:77} INFO - Job 154: Subtask ingest
[2022-02-12 17:36:55,128] {logging_mixin.py:109} INFO - Running <TaskInstance: test_dag.ingest scheduled__2021-01-02T06:00:00+00:00 [running]> on host c76e97d65bf1
[2022-02-12 17:36:55,199] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-02T06:00:00+00:00
[2022-02-12 17:36:55,200] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-12 17:36:55,201] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls -la /opt/***']
[2022-02-12 17:36:55,212] {subprocess.py:85} INFO - Output:
[2022-02-12 17:36:55,220] {subprocess.py:89} INFO - total 88
[2022-02-12 17:36:55,220] {subprocess.py:89} INFO - drwxrwxr-x 1 *** root  4096 Feb 12 17:36 .
[2022-02-12 17:36:55,220] {subprocess.py:89} INFO - drwxr-xr-x 1 root    root  4096 Jan 17 23:04 ..
[2022-02-12 17:36:55,221] {subprocess.py:89} INFO - -rw-rw-r-- 1 *** root 43931 Feb 12 17:33 ***.cfg
[2022-02-12 17:36:55,221] {subprocess.py:89} INFO - drwxrwxrwx 3 *** root  4096 Feb 12 13:03 dags
[2022-02-12 17:36:55,221] {subprocess.py:89} INFO - drwxrwxrwx 7 *** root  4096 Feb 12 13:19 logs
[2022-02-12 17:36:55,221] {subprocess.py:89} INFO - -rw-rw-r-- 1 *** root   308 Feb 12 17:36 output.csv
[2022-02-12 17:36:55,221] {subprocess.py:89} INFO - drwxrwxrwx 2 *** root  4096 Feb  7 13:15 plugins
[2022-02-12 17:36:55,221] {subprocess.py:89} INFO - -rwxrwxrwx 1 root    root    40 Feb  7 13:30 requirements.txt
[2022-02-12 17:36:55,221] {subprocess.py:89} INFO - drwxrwxrwx 2    1000 1000  4096 Feb  7 13:40 scripts
[2022-02-12 17:36:55,221] {subprocess.py:89} INFO - -rw-rw-r-- 1 *** root  4695 Feb 12 17:33 webserver_config.py
[2022-02-12 17:36:55,222] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-12 17:36:55,257] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=ingest, execution_date=20210102T060000, start_date=20220212T173654, end_date=20220212T173655
[2022-02-12 17:36:55,317] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-12 17:36:55,367] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
